fuel/src/main/kotlin/com/github/kittinunf/fuel/core/requests/DefaultRequest.kt

```diff
@@ -217,7 +218,7 @@ data class DefaultRequest(
      * @return [Request] the request
      */
     override fun body(stream: InputStream, calculateLength: BodyLength?, charset: Charset, repeatable: Boolean) =
-        body(openStream = { stream }, calculateLength = calculateLength, charset = charset, repeatable = repeatable)
+            body(openStream = { stream }, calculateLength = calculateLength, charset = charset, repeatable = repeatable)
```

Comment:
You probably didn't want to change this indentation to 8 instead of 4.

Reply:
Roger that